packages/app/src/cli/models/extensions/specifications/app_config_webhook.test.ts

```diff
@@ -20,13 +20,13 @@ describe('webhooks', () => {
             },
             {
               uri: 'pubsub://absolute-feat-test:pub-sub-topic2',
-              sub_topic: 'type:metaobject_one',
+              filter: 'type:metaobject_one',
```

Comment:
These tests seemed to be testing more than just sub_topic, so swapped them out with filter.